Sources/AWSSDKSwiftCore/Message/AWSRequest.swift

```diff
@@ -103,24 +115,26 @@ public struct AWSRequest {
         }
         return awsRequest
     }
+}
+
 
-    // Convert string to NIO HTTP Method
-    fileprivate func nioHTTPMethod(from: String) -> HTTPMethod {
+extension HTTPMethod {
```

Comment:
AFAIK, we can use the rawValue directly from SwiftNIO. Was released with `2.13.0`: https://github.com/apple/swift-nio/pull/1329

Reply:
done, plus upped the swift-nio version requirement in Package.swift

Reply:
Why do we still have this method here? Don't we want to use `HTTPMethod(rawValue: String)` directly?

Reply:
> Why do we still have this method here? Don't we want to use `HTTPMethod(rawValue: String)` directly?

We are using it. I ditched the old conversion method so we are forced to. Maybe github has got a little confused. Checkout the actual file